src/gui/mzroll/mainwindow.cpp

```diff
@@ -917,13 +917,6 @@ void MainWindow::showNotification(TableDockWidget* table) {
     connect(fluxomicsPrompt,
             SIGNAL(promptClicked()),
             SLOT(showPollyElmavenInterfaceDialog()));
-    connect(fluxomicsPrompt,
-            &Notificator::promptClicked,
-            this,
-            [=] {
-                analytics->hitEvent("Prompt", "Clicked", "PollyPhi");
-                pollyElmavenInterfaceDialog->switchToApp(PollyApp::PollyPhi);
-            });
     connect(fluxomicsPrompt,
             SIGNAL(promptClicked(TableDockWidget*)),
             pollyElmavenInterfaceDialog,
```

Comment:
Let's remove the entire `MainWindow::showNotification` method itself. It only ever gives a notification for PollyPhi. We do not need it anymore.